### ISSUE TITLE
The validation loss was large and almost unchanged, and the training loss has been decreasing

### ISSUE THREAD

**Issue by 20231211** [lyuwenyu/RT-DETR#277]
I ran the pytorch version of the code, and the code about the calculation loss was verified and uncommented, and the test_loss_bbox I got was very large, at 20000, but the train_loss_bbox was very small, and the train_loss kept dropping during training, but the AP would drop and rise, with a difference of nearly 0.02, it should be an overfit, and I wanted to output the verification loss, but test_loss_ bbox is big and has barely changed, I don't know what the reason is, how to fix it?
If I change the number of training rounds, do I need to adjust the learning rate?
Hope to get an answer, thank you!

**Comment by meirashaf** [comment]
having the same problem ☝🏻

**Comment by kir-7** [comment]
I was facing the same issue, for me val bbox loss reduced when I included ConvertBoxes transformation for val dataset, this transformation normalizes the boundig boxes values to between 0 and 1 so for train bbox loss I was getting small value, but for val bbox loss since the bbox values were not normalized it gave very high value.

**Comment by s5078345** [comment]
> I was facing the same issue, for me val bbox loss reduced when I included ConvertBoxes transformation for val dataset, this transformation normalizes the boundig boxes values to between 0 and 1 so for train bbox loss I was getting small value, but for val bbox loss since the bbox values were not normalized it gave very high value.

@kir-7 you're right.
I add following line in the [val dataloader](https://github.com/lyuwenyu/RT-DETR/blob/main/rtdetrv2_pytorch/configs/rtdetrv2/include/dataloader.yml):
```
- {type: ConvertBoxes, fmt: 'cxcywh', normalize: True}
```

Loss of test_loss_bbox decreases normally and my RT-DETR v2 model can detect things well.
